src/app/child-dev-project/notes/dashboard-widgets/no-recent-notes-dashboard/no-recent-notes-dashboard.component.ts

```diff
@@ -64,19 +86,27 @@ export class NoRecentNotesDashboardComponent implements OnInit {
       }
     }
 
-    this.concernedChildren = resultChildren.sort((a, b) => b.daysSinceLastNote - a.daysSinceLastNote);
+    this.concernedChildren = resultChildren.sort(
+      (a, b) => b.daysSinceLastNote - a.daysSinceLastNote,
+    );
 
     this.isLoading = false;
   }
 
-
   private isWithinDayRange(daysSinceLastNote: number) {
     if (this.fromBeginningOfWeek) {
-      return daysSinceLastNote <= (this.sinceDays + this.daysSinceBeginningOfWeek);
+      return (
+        daysSinceLastNote <= this.sinceDays + this.daysSinceBeginningOfWeek
+      );
     } else {
       return daysSinceLastNote <= this.sinceDays;
     }
   }
+
+  showChildDetails(child: ChildWithRecentNoteInfo) {
+    this.router.navigate(['/child', child.getId()]);
+  }
+
```

Comment:
We can use `[routerLink]="['/child', record.getId()]"` directly in the template in order to avoid the boilerplate function here in the component class.